spec/forms/password_form_spec.rb

```diff
@@ -12,7 +12,6 @@
     }
   end
 
-  it_behaves_like 'password validation'
```

Comment:
It's not clear to me that we'd want to remove these validations? (Same comment for file below `spec/forms/update_user_password_form_spec.rb`)